packages/svelte/src/internal/client/runtime.js

```diff
@@ -162,9 +162,9 @@ export function check_dirtiness(reaction) {
 
 	if ((flags & MAYBE_DIRTY) !== 0) {
 		var dependencies = reaction.deps;
+		var is_unowned = (flags & UNOWNED) !== 0;
```

Comment:
no need to create a single-use variable

```suggestion
```

Reply:
It's used more than once though?

Reply:
ah my bad, it looked otherwise from the diff but in context yep